File: examples/TeamRemoveMemberExample.py
import json
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    username="YOUR_API_KEY",
    # access_token="YOUR_ACCESS_TOKEN",
)

with ApiClient(configuration) as api_client:
    team_remove_member_request = models.TeamRemoveMemberRequest(
        email_address="teammate@dropboxsign.com",
        new_owner_email_address="new_teammate@dropboxsign.com",
    )

    try:
        response = api.TeamApi(api_client).team_remove_member(
            team_remove_member_request=team_remove_member_request,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling TeamApi#team_remove_member: %s\n" % e)

File: examples/TeamRemoveMemberExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class TeamRemoveMemberExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.Username = "YOUR_API_KEY";
        // config.AccessToken = "YOUR_ACCESS_TOKEN";

        var teamRemoveMemberRequest = new TeamRemoveMemberRequest(
            emailAddress: "teammate@dropboxsign.com",
            newOwnerEmailAddress: "new_teammate@dropboxsign.com"
        );

        try
        {
            var response = new TeamApi(config).TeamRemoveMember(
                teamRemoveMemberRequest: teamRemoveMemberRequest
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling TeamApi#TeamRemoveMember: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
